bitcoin/electrs/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM rust:alpine3.13
+
+RUN apk add clang-dev build-base linux-headers git
```

`--no-cache` flag is recommended to keep the image smaller.

```suggestion
RUN apk add --no-cache clang-dev build-base linux-headers git
```